Refs #31055 -- Doc'd 'databases' argument of check functions. (#13771)
django/django: docs/topics/checks.txt

The check function *must* accept an ``app_configs`` argument; this argument is
the list of applications that should be inspected. If ``None``, the check must
be run on *all* installed apps in the project.

> The check will also receive a ``databases`` argument, which is a list of
> database aliases whose connections may be used to inspect database level
> configuration. If this is ``None``, the check must not use any database
> connections.

Reviewer: ```suggestion
The check will receive a ``databases`` keyword argument. This is a list of
database aliases whose connections may be used to inspect database level
configuration. If ``databases`` is ``None``, the check must not use any
database connections.
```
